metaflow/plugins/argo/test/test_trigger_live_run.py

```diff
@@ -0,0 +1,265 @@
+from metaflow.client.trigger_live_run import trigger_live_run
+from metaflow.exception import MetaflowException
+import time
+from unittest import TestCase
+
+
+"""
+Tests Outline:
+    - Parameter Passing (flow name "LiveRunParamFlow")
+        - successful parameter passing
+        - wrong type parameter passing (eg, type=set)
+    - Flow Identification (flow name "LiveRunSimpleFlow")
+        - id by flow name
+        - id by template name
+        - id by both (matching)
+        - id by both (mismatch)
+        - id by neither flow name nor template name 
+    - Plugin Name (all other tests correctly use 'Argo')
+        - None
+        - 'FakePlugin'
+    - Other Tests
+        - timeout error (flow takes too long - set wait_timeout to 0)
+        - failure flow (flow name "LiveRunFailureFlow")
+        - missing flow (flow name "LiveRunMissingFlow")
+        - flow that triggers a flow (flow name "LiveRunFlowTriggeringFlow")
+    - Features/aspects tested but w/o their own named tests
+        - Wait Feature - no separate tests because both states (True & False)
+            are incorporated into Flow Identification section
+"""
+
+
+def start_message(name):
+    print(f"\nStarting test '{name}'")
+
+
+def finished_message(name):
+    print(f"Finished test '{name}'")
+
+
+test_case = TestCase()
```

Comment:
Could you please restructure this file according to sample unit test usage here? https://docs.python.org/3/library/unittest.html#basic-example

Eventually we'd like to have some unit test package (for example `pytest`) running your test scripts.